src/pages/workspaces/workspace/WorkspaceMenu.js

```diff
@@ -21,7 +21,7 @@ const WorkspaceMenu = ({
 
   const menuContent = !!namespace ?
     h(DynamicWorkspaceMenuContent, { namespace, name, onShare, onClone, onDelete, onLock }) :
-    h(WorkspaceMenuContent, { canShare, isAzureWorkspace, isLocked, isOwner, onClone, onShare, onLock, onDelete })
+    h(WorkspaceMenuContent, { canShare, isAzureWorkspace, isLocked, isOwner, onClone, onShare, onLock, onDelete, workspaceLoaded })
 
   return h(MenuTrigger, {
```

Comment:
@breilly2 the thing is that the workspace details aren't fetched until you actually click on the menu icon… we don't want to get the details for all the workspaces since there may be a very large number.

I don't see the behavior with clone that you are describing (initially enabled) on the PR site.